src/stillir.app.src

```diff
@@ -2,8 +2,11 @@
 {application, stillir,
  [
   {description, "Read Unix env vars into application config"},
-  {vsn, "1.0.0"},
+  {vsn, git},
   {registered, []},
   {applications, [kernel,
-                  stdlib]}
+                  stdlib]},
+  {maintainers, ["Fred Hebert"]},
+  {licenses, ["BSD"]},
```

Comment:
I believe hex.pm will automatically extract this information from the `LICENSE` file.

Reply:
Hex documentation says that the `licenses` field is required: https://hex.pm/docs/rebar3_publish